blog.html

```diff
@@ -22,6 +22,286 @@
+                        were made in Kafka 4.0 to enable users to view all groups in a cluster, along with their types and
+                        protocols, providing accurate information even when the Admin Client API fails.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/w4mMEw">KIP-1099: Extend kafka-consumer-groups command line tool to support new consumer group</a>
+                        <br>
+                        This KIP extends the kafka-consumer-groups and the kafka-share-groups command line tools to
+                        provide more information related to consumer and share groups. This helps when it comes to
+                        troubleshooting consumer and share groups, respectively introduced by KIP-848 and KIP-932.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/KYuMEw">KIP-1102: Enable clients to rebootstrap based on timeout or error code</a>
+                        <br>
+                        KIP-1102 enhances Kafka client resilience by proactively triggering metadata rebootstrap when no
+                        updates occur within a timeout period and allowing servers to explicitly signal clients to rebootstrap.
+                        This addresses previous limitations in KIP-899, where clients could become stuck with outdated
+                        metadata unless all brokers were unreachable.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/K5sODg">KIP-896: Remove old client protocol API versions in Kafka 4.0</a>
+                        <br>
+                        For the first time, old protocol API versions have been removed. Users should ensure brokers are
+                        version 2.1 or higher before upgrading Java clients (including Connect and Streams to 4.0. Similarly,
+                        users should ensure their Java clients version is 2.1 or higher before upgrading brokers to 4.0.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/y4kgF">KIP-1124: Providing a clear Kafka Client upgrade path for 4.x</a>
+                        <br>
+                        This KIP outlines the upgrade patch for Kafka Clients, Streams and Connect to 4.0. It is a must read
+                        before upgrading your clusters and clients.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/GCx4CQ">KIP-653: Upgrade log4j to log4j2</a>
+                        <br>
+                        The logging framework has been migrated from Log4j to Log4j2. Users can use the
+                        log4j-transform-cli tool to automatically convert their existing Log4j configuration files to Log4j2
+                        format.
```

Comment:
Perhaps we could remind users that Log4j 1.x configurations are still acceptable, but with certain limitations, as outlined in the migration guide (https://logging.apache.org/log4j/2.x/migrate-from-log4j1.html#limitations-of-the-log4j-1-x-bridge)

Reply:
+1